notebooks/a00_introduction_to_geomstats.ipynb

```diff
@@ -0,0 +1,290 @@
+{
```

Comment:
<p>We say: "Many datasets lie on a manifold", but we have not introduced a manifold yet. (since it will be done in the next notebook).</p><p> </p><p>Should we still say something like: “the definition of manifold will be introduced later, but for now we can see it as a nonlinear generalization of a vector space, such as the surface in the figure below [add one more beautiful drawing?]”</p><p> </p><p>1) making computations less complicated --&gt; making computations more intuitive and interpretable.</p><p>(Some could argue that Riemannian operations are quite complicated!)</p><p> </p><p>2) predictive power is good actually!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1304/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>